### PR TITLE
Example type dumping from amino

### PR DESCRIPTION
**Pull request by jackzampolin**
This is the result of a conversation with @aaronc and @hschoenburg about amino types in JS

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -19,7 +19,7 @@ require (
 	github.com/spf13/cobra v0.0.5
 	github.com/spf13/viper v1.4.0
 	github.com/stretchr/testify v1.4.0
-	github.com/tendermint/go-amino v0.15.0
+	github.com/tendermint/go-amino v0.15.1-0.20191003185523-ddfc7fca5e86
```
Comment (hschoenburg) [inline go.mod:22]: Just pointing out this is the key to accessing the typeInfo's of amino

Comment (alexanderbez) [reply go.mod:22]: Is this officially released?

**Review by alexanderbez — CHANGES_REQUESTED** [review]

File: go.mod (hunk 1 above)
Comment (hschoenburg) [reply go.mod:22]: its just a demo jacked whipped up for use internally so we can use it in building out the typescript libs

Comment (hschoenburg) [reply go.mod:22]: https://github.com/cosmos/gaia/pull/150/files

**Comment by aaronc** [comment]
Has there been any follow up on this? Is it being actively worked on somewhere or in the backlog?

**Pull request closed without merging**